control-base/src/main/java/fi/nls/oskari/control/data/GetPrintHandler.java

```diff
@@ -118,19 +120,44 @@ private PrintRequest createPrintRequest(ActionParameters params)
         request.setShowLogo(params.getHttpParam(PARM_LOGO, false));
         request.setShowScale(params.getHttpParam(PARM_SCALE, false));
         request.setShowDate(params.getHttpParam(PARM_DATE, false));
-
+        request.setScaleText(params.getHttpParam(PARM_SCALE_TEXT, ""));
         setPagesize(params, request);
         setCoordinates(params.getRequiredParam(PARM_COORD), request);
         setFormat(params.getRequiredParam(PARM_FORMAT), request);
+        request.setUseConfiguredScale(params.getHttpParam(PARM_USE_CONFIGURED_SCALE, false));
 
         List<PrintLayer> layers = getLayers(params.getRequiredParam(PARM_MAPLAYERS),
                 params.getUser());
+        layers = filterLayersWithZeroOpacity(layers);
```

Comment:
Layers with 0 opacity are already filtered out in getLayers: https://github.com/dimenteq/oskari-server/blob/7b798ba64ce0e35dfe9ddab56f8f2dd2dbd8bd31/control-base/src/main/java/fi/nls/oskari/control/data/GetPrintHandler.java#L282

So this can be removed.